scripts/base_tools.sh

```diff
@@ -115,6 +77,7 @@ as_root apt-get install -y --no-install-recommends \
         iputils-ping \
         jq \
         make \
+        python \
```

Comment:
What still needs python?

Reply:
It must be something in the sel4test chain somewhere, because this failed really early on. We're not installing any dependencies for python 2, so it's probably something quite simple, but I didn't want to extend this PR to hunt down all of these and update them to python 3.

It would be a good second step, though, can't be much left any more.

Reply:
Might be worth seeing if installing [python-is-python3](https://packages.debian.org/bullseye/python-is-python3) just fixes things.

Reply:
I was too chicken to do that for now, because that might change the behaviour in subtle ways. But for a second step, I'd be up for it.